tests/string_concat/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,58 @@
+exports[`test string_concat.js 1`] = `
+"var fnString =
+  \'\"\' + this.USE + \' \' + this.STRICT + \'\";\\n\' +
+  this.filterPrefix() +
+  \'var fn=\' + this.generateFunction(\'fn\', \'s,l,a,i\') +
+  extra +
+  this.watchFns() +
+  \'return fn;\';
+
+console.error(
+  \'In order to use \' + prompt + \', you need to configure a \' +
+  \'few environment variables to be able to commit to the \' +
+  \'repository. Follow those steps to get you setup:\\n\' +
+  \'\\n\' +
+  \'Go to https://github.com/settings/tokens/new\\n\' +
+  \' - Fill \"Token description\" with \"\' + prompt + \' for \' +
+    repoSlug + \'\"\\n\' +
+  \' - Check \"public_repo\"\\n\' +
+  \' - Press \"Generate Token\"\\n\' +
+  \'\\n\' +
+  \'In a different tab, go to https://travis-ci.org/\' +
+    repoSlug + \'/settings\\n\' +
+  \' - Make sure \"Build only if .travis.yml is present\" is ON\\n\' +
+  \' - Fill \"Name\" with \"GITHUB_USER\" and \"Value\" with the name of the \' +
+    \'account you generated the token with. Press \"Add\"\\n\' +
+  \'\\n\' +
+  \'Once this is done, commit anything to the repository to restart \' +
+    \'Travis and it should work :)\'
+);
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+var fnString = \'\"\' + this.USE + \" \" + this.STRICT + \'\";\\n\' +
+  this.filterPrefix() +
+  \"var fn=\" + this.generateFunction(\"fn\", \"s,l,a,i\") +
+  extra + this.watchFns() +
```

Comment:
Notice how `extra` doesn't have a string literal around so it isn't on its own line.